examples/common/stopwatch.h

```diff
@@ -25,7 +25,7 @@
 #ifndef STOPWATCH_H
 #define STOPWATCH_H
 
-#if (_WIN32 || _WIN64)
+#if (_WIN32)
```

Comment:
`_WIN32` is always defined on Windows, even for 64bit builds.

https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?redirectedfrom=MSDN&view=msvc-170